Ladybird/WebContentView.h

```diff
@@ -68,7 +68,6 @@ class WebContentView final
     Function<void(const AK::URL&)> on_load_start;
     Function<void(const AK::URL&)> on_load_finish;
     Function<void(Gfx::Bitmap const&)> on_favicon_change;
-    Function<void(const AK::URL&)> on_url_drop;
```

Comment:
Why is this removed?

Reply:
it wasn't used at all (like most other Function<> members here) and it looks like it was copy-pasted from LibWebView.

<details>

https://github.com/SerenityOS/serenity/blob/aee7c44064b4ad24709e3b5c5b5bb63b3b2d7354/Userland/Libraries/LibWebView/OutOfProcessWebView.h#L83-L109

https://github.com/SerenityOS/serenity/blob/aee7c44064b4ad24709e3b5c5b5bb63b3b2d7354/Ladybird/WebContentView.h#L61-L83

</details>

so i just removed it to avoid confusion

Reply:
I'd prefer if we deal with those separately instead of removing one of many unused ones now :)